Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -655,6 +655,9 @@ def test_parse_autonomous_system_indicator(self, taxii_2_client):
 
         When:
          - parsing the autonomous-system into a format XSOAR knows to read.
+           1. update_custom_fields = False
+           2. update_custom_fields = True
+
 
         Then:
          - make sure all the fields are being parsed correctly.
```

Comment:
Let's add to the "Then" part what are you expecting in case 1 and in case 2 (no custom fields to appear in the indicator and vise versa)